src/endpoints/events.js

```diff
@@ -0,0 +1,42 @@
+import Endpoint from './Endpoint'
+import EventModel from './models/EventModel'
+import DateModel from './models/DateModel'
+
+const BIRTH_OF_UNIVERSE = new Date(0).toISOString().split('.')[0] + 'Z'
+
+export default new Endpoint({
+  name: 'events',
+  url: 'https://cms.integreat-app.de/{location}/{language}/wp-json/extensions/v0/modified_content/events?since={since}',
+  jsonToAny: (json) => {
+    if (!json) {
+      return []
+    }
+
+    const parseDate = (date, time) => date ? new Date(date + 'T' + (time || '00:00:00') + 'Z') : null
+
+    json = json.filter(event => event.status === 'publish')
+    json = json.map(event => new EventModel({
+      id: event.id,
+      title: event.title,
+      content: event.content,
+      thumbnail: event.thumbnail,
+      address: event.location.address,
+      town: event.location.town,
+      date: new DateModel({
+        startDate: parseDate(event.event.start_date, event.event.start_time),
+        endDate: parseDate(event.event.end_date, event.event.end_time),
+        allDay: event.event.all_day === '0'
+      })
+    }))
+    json = json.filter(event => !event.date.startDate || event.date.startDate > Date.now() - 1000 * 60 * 60 * 24)
```

Comment:
What is this magic number here? Is this documented and exptected?

Reply:
The magic number is.... drumrolls please.... 🥁 🥁 🥁 .... Milliseconds of a day
(Just so an events is displayed a little bit longer than until it starts - there are also few enough events to do that usually :D)